content/src/filings/annual-report.md

```diff
@@ -19,13 +19,16 @@ callToActionText: File Your Annual Report
 - Names and addresses of all business partners
 - Applicable fees paid
 
-:::callout{ header="" icon="true" calloutType="conditional" }
+:::callout{ showHeader="true" headerText="" showIcon="true" calloutType="conditional" }
 
 - Confirmation of submission of your annual report
-  :::
 
-:::callout{ header="What if I do not complete my report?" icon="true" calloutType="conditional" }
+:::
+
+:::callout{ showHeader="true" headerText="What if I do not complete my report?" showIcon="false" calloutType="conditional" }
```

Comment:
pebble - how did showIcon change?

Reply:
That's a great question. I don't know. I'll fix it.